Resources/Prototypes/_Sunrise/Chemistry/medicine.yml

```
@@ -0,0 +1,24 @@
+- type: reagent
+  id: HemostaticPowder
+  name: reagent-name-hemostatic-powder
+  group: Medicine
+  desc: reagent-desc-hemostatic-powder
+  physicalDesc: reagent-physical-desc-grainy
+  flavor: medicine
+  color: "#8b0b47"
+  metabolisms:
+    Medicine:
+      effects:
+      - !type:ModifyBleedAmount
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)